packages/app/src/pages/endpoints/[endpointId]/_/content/parts/body/index.tsx

```diff
@@ -50,7 +50,11 @@ const Body: React.FC<Props> = ({
 
   const elm = useMemo<JSX.Element>(() => {
     if (base.isPending) {
-      return <Spinner className="w-4" on={COLOR_SYSTEM.SURFACE} />;
+      return (
+        <div className="p-10 flex items-center justify-center">
```

Comment:
Since there is no reasonable way to determine the height, give an appropriate space that is not too tight.